site/content/docs/contributing/getting-started.md

```diff
@@ -85,22 +91,27 @@ if there is any existing discussion or work related to your interests.
 
 If you do not see anything, please [file a new issue][file an issue].
 
+> **NOTE**: _Please_ file an enhancement / [feature request issue][fr-issue] to discuss features before filing a PR (ideally even before writing any code), we have a lot to consider with respect to our
+> existing users and future support when accepting any new feature.
+> To streamline the process, please reach out and discuss the concept and design
+> / approach ASAP so the maintainers and community can get involved early.
+
 Please reach out for bugs, feature requests, and other issues!  
 The maintainers of this project are reachable via:
 
-- [Kubernetes Slack] in the [#kind] channel
-- [filing an issue][file an issue]
-- The Kubernetes [SIG-Testing Mailing List]
+- [Kubernetes Slack] in the [#kind] channel (most active, along with the community)
+- The issue tracker by [filing an issue][file an issue]
+- The Kubernetes [SIG-Testing][SIG-Testing] [Mailing List][SIG-Testing Mailing List]
```

Comment:
is this a copy paste issue or is the correct format? is not `[SIG-Testing][SIG-Testing Mailing List]`

Reply:
this is now two links.

Reply:
(check the rendered form in the preview)